server/boot/model-explorer.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function mountLoopBackExplorer(server) {
+  var explorer;
+  try {
+    explorer = require('loopback-explorer');
+    var restApiRoot = server.get('restApiRoot');
+
+    var explorerApp = explorer(server, {basePath: restApiRoot});
+    server.use('/explorer', explorerApp);
```

Comment:
I still need to clarify with @ritch on this. Im not sure what happens to /explorer and if it supports exposing models from sub-components of the parent app.